telegram/ext/jobqueue.py

```diff
@@ -70,30 +71,25 @@ def __init__(self):
     def set_dispatcher(self, dispatcher):
         self._dispatcher = dispatcher
 
-    def _put(self, job, next_t=None, last_t=None):
-        if next_t is None:
-            next_t = job.interval
-            if next_t is None:
-                raise ValueError('next_t is None')
-
-        if isinstance(next_t, datetime.datetime):
-            next_t = (next_t - datetime.datetime.now()).total_seconds()
-
-        elif isinstance(next_t, datetime.time):
-            next_datetime = datetime.datetime.combine(datetime.date.today(), next_t)
-
-            if datetime.datetime.now().time() > next_t:
-                next_datetime += datetime.timedelta(days=1)
-
-            next_t = (next_datetime - datetime.datetime.now()).total_seconds()
+    def _put(self, job, next_t=None, previous_t=None):
+        """
+        Enqueues the job, scheduling its next run at the correct time.
 
-        elif isinstance(next_t, datetime.timedelta):
-            next_t = next_t.total_seconds()
+        Args:
+            job (telegram.ext.Job): job to enqueue
+            next_t (optional):
+                Time for which the job should be scheduled. The precise semantics of this parameter
+                depend on its type (see :func:`telegram.ext.JobQueue.run_repeating` for details).
+                Defaults to now + ``job.interval``.
+            previous_t (optional):
+                Time at which the job last ran (``None`` if it hasn't run yet).
+        """
 
```

Comment:
Please move the empty line to before the `"""`.

Reply:
Fixed in 6ed29bb . I actually didn't know this was a PEP 257 thing! I've been doing my docstrings wrong all along... :laugh: